server/routes/createGoal/index.ts

```diff
@@ -14,8 +16,44 @@ import {
  * Route definitions for the pages relating to Creating A Goal
  */
 export default (router: Router, services: Services) => {
-  const createGoalController = new CreateGoalController(services.educationAndWorkPlanService)
+  if (config.featureToggles.newCreateGoalRoutesEnabled) {
```

Comment:
Using the new feature toggle, we either setup routes with the new index based paths using the controller; or setup the existing routes using the original behaviour controller.